frontend/src/constants/dataConstants.js

```diff
@@ -131,13 +160,140 @@ export const DATA_CONTINUOUS_SUMMARY = "CS",
                         16.7743, 20.0571, 15.1564, 15.0368
                     ],
                 };
-                /* eslint-enable */
+            /* eslint-enable */
```

Comment:
use the  eslint enable/disable to skip linting; like what was done for continuous individual

make one line for each dose of the same value. So there should be 3 lines for each doses, 3 lines for litter_n, etc.